reactive/calico.py

```diff
@@ -291,6 +291,13 @@ def install_calico_service():
     ip_versions = {net.version for net in get_networks(charm_config('cidr'))}
     ip4 = get_bind_address() if 4 in ip_versions else "none"
     ip6 = "autodetect" if 6 in ip_versions else "none"
+
+    if not os.path.exists('/opt/calicoctl/kubeconfig'):
+        with open('/root/.kube/config') as src:
+            data = src.read()
+            with open('/opt/calicoctl/kubeconfig') as dst:
+                dst.write(data)
```

Comment:
This will give calico-node admin access to the Kubernetes cluster. I think that's probably OK for a temporary charm.

Reply:
Confirmed.